src/pre_defined_classes/simple_programs_taking_numeric_user_input/medium/q7/MainTest.java

```diff
@@ -0,0 +1,47 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.medium.q7;
+
+
+/*
+Sample output:
+Enter your score:
+-5
+Your goal is 45. Keep up the good work!
+ */
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Random;
+import java.util.stream.Stream;
+
+public class MainTest extends BaseTest {
```

Comment:
This should extend BaseRandomTest so you can use the RandomInteger() clause instead